Dockerfile

```diff
@@ -1,24 +1,47 @@
 FROM node:lts-alpine
+
+ARG VERSION_INFO=noVersion
+
 ENV NPM_CONFIG_LOGLEVEL warn
+ENV PORT=1337
 
-# Install "git"
```

Comment:
Installing `git` was apparently done only to allow running `git describe --tags` in `config/version.js`. This is slow and bloats the size of the image, and is IMHO unnecessary just to get a commit hash in the footer.